perception/autoware_ground_segmentation/src/ransac_ground_filter/node.hpp

```diff
@@ -16,6 +16,7 @@
 #define RANSAC_GROUND_FILTER__NODE_HPP_
 
 #include "autoware/pointcloud_preprocessor/filter.hpp"
+#include "autoware/universe_utils/ros/static_transform_buffer.hpp"
```

Comment:
I got confused with headers under the new namespaces

https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/languages/cpp/

Should `universe_utils` use `<>` or `""` ?

Reply:
You can check compiler search part via `g++ -v -E -x c++ - < /dev/null`. My output (in docker image) doesn't consider any of Autoware package to be placed in `<>`.

Reply:
Let's follow Autoware guidelines
Addressed in https://github.com/autowarefoundation/autoware.universe/pull/8174/commits/69eb78a52e15dae322ce417b4d85ddbfed9a8b38